website/content/about/_index.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Did you leave this in thinking we'd use the About section for TAG App Delivery content after merge?

Reply:
It's just placeholder content and a suggestion for the type of content you may want here. We can remove if not needed at this point.